src/settings/_box-style.scss

```diff
@@ -3,3 +3,7 @@
 
 // Border radius
 $bdrs-m: 2px !default;
+
+// Form focus
+$bxsh-focus: 0 0 3px 0 color-variation($c-primary, 1) !default;
```

Comment:
I think we need a $bxsh-m (l,xl) more that something specific to focus (input).
This definition should be submitted to UXers.

Reply:
Check glow or outline